src/tbb/governor.cpp

```diff
@@ -367,15 +367,18 @@ static void (*restore_affinity_ptr)( binding_handler* handler_ptr, int slot_num
 int (*get_default_concurrency_ptr)( int numa_id, int core_type_id, int max_threads_per_core )
     = dummy_get_default_concurrency;
 
-#if _WIN32 || _WIN64 || __unix__
+#if _WIN32 || _WIN64 || __unix__ || __APPLE__
+
 // Table describing how to link the handlers.
 static const dynamic_link_descriptor TbbBindLinkTable[] = {
     DLD(__TBB_internal_initialize_system_topology, initialize_system_topology_ptr),
     DLD(__TBB_internal_destroy_system_topology, destroy_system_topology_ptr),
+#if __TBB_CPUBIND_PRESENT
     DLD(__TBB_internal_allocate_binding_handler, allocate_binding_handler_ptr),
     DLD(__TBB_internal_deallocate_binding_handler, deallocate_binding_handler_ptr),
     DLD(__TBB_internal_apply_affinity, apply_affinity_ptr),
     DLD(__TBB_internal_restore_affinity, restore_affinity_ptr),
+#endif
```

Comment:
can it be dynamically understood that tbbbind does not have all symbols and "bind"-related symbols are just not set upon loading of tbbbind?

Reply:
`dynamic_link` functionality guarantees that either all of requested symbols are loaded, or none of them, if something is missing. Thus, we shouldn't expect hwloc's cpubind related symbols here in the first place.